ansible/group_vars/all/inv_mapping.yml

```diff
@@ -0,0 +1,4 @@
+---
+# Specify all internal mappings below
+lab: lab_connection_graph.xml
+inventory: lab_connection_graph.xml
```

Comment:
Please add 3 line gap after inventory with comments. That way creates nice separation for any internal changes after the gap.

Reply:
Fixed